kernel/src/threading/scheduling/mod.rs

```diff
@@ -0,0 +1,38 @@
+pub mod fifo_scheduler;
+pub mod scheduler;
```

Comment:
Same advice here about removing `pub` and intentionally `pub use`'ing things that need to be accessible elsewhere.

```suggestion
mod fifo_scheduler;
mod scheduler;
```